.gitignore

```diff
@@ -8,3 +8,4 @@ pom.xml.asc
 .lein-*
 .nrepl-port
 reports
+*.cpcache
```

Comment:
I don't know if I need this. The contribution style guide says no but this gets added as a change for me - so I added it.

Reply:
Pull requests should only contain changes specific to the feature being implemented, so this should be removed.